content/docs/ui/sending-email/working-with-marketing-templates.md

```diff
@@ -76,6 +76,6 @@ You can easily duplicate a pre-built template provided by SendGrid.
 
  ## 	Additional Resources
 
-- [Sending an Email]({{root_url}}/ui/sending-email/how-to-send-email-with-marketing-campaigns/)
-- [Design Editor]({{root_url}}/ui/sending-email/editor/#-The-Design-Editor)
-- [Code Editor]({{root_url}}/ui/sending-email/editor/#the-code-editor)
```

Comment:
@ashishpatelcs why did you decide that these are broken links? They work great on https://sendgrid.com/docs/ui/sending-email/working-with-marketing-templates/ .
The `root_url` word will be replaced with the correct prefix when building the project.